src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck start work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck stop work");
```

Comment:
Similarly, the message 'Truck stop work' should be more informative. It should clearly indicate that the Truck has stopped working and, if possible, provide additional context related to the truck's operations.